### PR TITLE
Fixing SizeF, CookieContainer, CookieCollection serialization between Core <--> Desktop and adding full binary mode tests

### PR DESCRIPTION
**Pull request by ViktorHofer**
https://github.com/dotnet/corefx/pull/21223

Requires https://github.com/dotnet/coreclr/pull/12367

cc @danmosemsft

### CODE REVIEW

**Comment by danmosemsft** [comment]
Updated to match the master change.

File: src/System.Net.Primitives/src/System/Net/CookieContainer.cs [hunk 1]
```diff
@@ -1048,6 +1048,7 @@ public IEnumerator GetEnumerator()
         public object SyncRoot => m_list.SyncRoot;
 
         [Serializable]
+        [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```
Comment (krwq) [inline src/System.Net.Primitives/src/System/Net/CookieContainer.cs]: Should this be b03f5f7f11d50a3a?

**Comment by ViktorHofer** [comment]
Approved by shiproom. After new coreclr version is in release/2.0.0 branch I will take a look @krwq. Thanks!

**Comment by krwq** [comment]
@ViktorHofer could you take a look at the PublicKeyToken comment in src/System.Net.Primitives/src/System/Net/CookieContainer.cs, you got more context on this

File: src/System.Net.Primitives/src/System/Net/CookieContainer.cs (hunk 1 above)
Comment (morganbr) [inline src/System.Net.Primitives/src/System/Net/CookieContainer.cs]: b77 is correct

**Comment by krwq** [comment]
@morganbr thanks for confirming!

@danmosemsft this is good to go assuming this is approved

**Comment by ViktorHofer** [comment]
The thing is, the coreclr change isn't yet in. Without comparing blobs we have green CI but in fact it shouldn't be green... Waiting just to make sure

**Comment by krwq** [comment]
@ViktorHofer what scenarios are broken without coreclr changes?

**Comment by ViktorHofer** [comment]
Serialization with full mode of types which depend on CompareInfo which itself depends on the introduced culture field. Should at least be a couple of types. Because we already updated the blobs and don't have a equals test for this field CI is green although it shouldn't be till the corevlr bits are in

**Comment by danmosemsft** [comment]
We have a new CLR. 
@dotnet-bot test this plesae

**Comment by danmosemsft** [comment]
@dotnet-bot test this please (spell please right)

**Comment by danmosemsft** [comment]
@viktorhofer good to merge?

**Pull request merged**